cypress/e2e/add-liquidity.test.ts

```diff
@@ -3,6 +3,10 @@ import { CyHttpMessages } from 'cypress/types/net-stubbing'
 import { aliasQuery, hasQuery } from '../utils/graphql-test-utils'
 
 describe('Add Liquidity', () => {
+  before(() => {
+    window.localStorage.set('selectedWallet', 'INJECTED')
```

Comment:
I think we want this to be a part of the visit override, and not done in each individual test. See cypress/support/e2e.ts: `Cypress.Commands.overwrite('visit', ...)`.